docs/installation.md

```diff
@@ -182,10 +177,12 @@ Get any errors? [See our troubleshooting tips.](#troubleshooting)
   (a way of running software in an isolated environment) and "images" (a
   snapshot of all of the files neccessary to run a container).
 - **Docker Compose**: Compose allows you to configure and run a collection of
-  connected containers (like a web application and it's database)
-- **Docker Machine**: Docker only runs natively on Linux and Windows. On OS X,
-  we'll use Docker Machine to start the Docker server in a virtual linux
-  environment (using Virtualbox)
+  connected containers (like a web application and its database)
+- **Docker Machine**: In environments where Docker Engine is not supported, or
+  where it is not approved for use, Docker Machine can be used to create and
```

Comment:
Perhaps "or where it is not available" is a more general way to describe this. :smile: